Source: Ali Polatel <polatel@gmail.com>
Upstream: No
Reason: Disable tests that fail with sydbox (reasons below)

Edit: Daniel Mierswa<impulze@impulze.org>
I removed the disabling of 2 tests here since they seem to work fine
with 7.5. There's another test failing on my machine (tail-2/assert)
but since it works for alip, let's see if more people run into it.

diff -ur coreutils-7.4/tests/ls/stat-free-symlinks coreutils-7.4-sydbox/tests/ls/stat-free-symlinks
--- coreutils-7.4/tests/ls/stat-free-symlinks	2009-02-27 18:36:00.000000000 +0200
+++ coreutils-7.4-sydbox/tests/ls/stat-free-symlinks	2009-05-21 15:48:50.000000000 +0300
@@ -24,6 +24,8 @@
 . $srcdir/test-lib.sh
 require_strace_ stat

+skip_test_ 'This test fails with sydbox due to strace usage'
+
 touch x || framework_failure
 chmod a+x x || framework_failure
 ln -s x link-to-x || framework_failure
diff -ur coreutils-7.4/tests/mv/atomic coreutils-7.4-sydbox/tests/mv/atomic
--- coreutils-7.4/tests/mv/atomic	2009-02-27 18:36:01.000000000 +0200
+++ coreutils-7.4-sydbox/tests/mv/atomic	2009-05-21 15:49:56.000000000 +0300
@@ -24,6 +24,8 @@
 . $srcdir/test-lib.sh
 require_strace_ unlink

+skip_test_ 'This test fails with sydbox due to strace usage'
+
 # Before the fix, mv would unnecessarily unlink the destination symlink:
 #   $ rm -rf s[12]; ln -s / s1; ln -s /tmp s2; strace -qe unlink /bin/mv -T s1 s2
 #   unlink("s2") = 0
diff -ur coreutils-7.4/tests/mv/atomic2 coreutils-7.4-sydbox/tests/mv/atomic2
--- coreutils-7.4/tests/mv/atomic2	2009-02-27 18:36:01.000000000 +0200
+++ coreutils-7.4-sydbox/tests/mv/atomic2	2009-05-21 15:49:22.000000000 +0300
@@ -24,6 +24,8 @@
 . $srcdir/test-lib.sh
 require_strace_ unlink

+skip_test_ 'This test fails with sydbox due to strace usage'
+
 # Before the fix, mv would unnecessarily unlink the destination symlink:
 #   $ rm -f a b b2; touch a b; ln b b2; strace -e unlink /p/bin/mv a b
 #   unlink("b")                             = 0
diff -ur coreutils-7.4/tests/install/basic-1 coreutils-7.4-sydbox/tests/install/basic-1
--- coreutils-7.4/tests/install/basic-1	2009-02-27 18:36:00.000000000 +0200
+++ coreutils-7.4-sydbox/tests/install/basic-1	2009-08-16 18:54:49.000000000 +0300
@@ -106,15 +106,16 @@
 # inaccessible parent.  coreutils 5.97 fails this test.
 # Perform this test only if "." is on a local file system.
 # Otherwise, it would fail e.g., on an NFS-mounted file system.
-if df --local . >/dev/null 2>&1; then
-  mkdir -p sub1/d || fail=1
-  (cd sub1/d && chmod a-r . && chmod a-rx .. &&
-   ginstall -d "$iwd/xx/zz" rel/a rel/b) || fail=1
-  chmod 755 sub1 sub1/d || fail=1
-  test -d xx/zz || fail=1
-  test -d sub1/d/rel/a || fail=1
-  test -d sub1/d/rel/b || fail=1
-fi
+# XXX: This test fails with sydbox as well, see my comment in bug 204. -alip
+#if df --local . >/dev/null 2>&1; then
+#  mkdir -p sub1/d || fail=1
+#  (cd sub1/d && chmod a-r . && chmod a-rx .. &&
+#   ginstall -d "$iwd/xx/zz" rel/a rel/b) || fail=1
+#  chmod 755 sub1 sub1/d || fail=1
+#  test -d xx/zz || fail=1
+#  test -d sub1/d/rel/a || fail=1
+#  test -d sub1/d/rel/b || fail=1
+#fi
 
 touch file || fail=1
 ginstall -Dv file sub3/a/b/c/file >out 2>&1 || fail=1
diff -ur coreutils-7.4/tests/mkdir/p-3 coreutils-7.4-sydbox/tests/mkdir/p-3
--- coreutils-7.4/tests/mkdir/p-3	2009-02-27 18:36:01.000000000 +0200
+++ coreutils-7.4-sydbox/tests/mkdir/p-3	2009-08-16 18:55:21.000000000 +0300
@@ -42,11 +42,12 @@
 # whose parent is inaccessible.  coreutils 5.97 fails this test.
 # Perform this test only if "." is on a local file system.
 # Otherwise, it would fail e.g., on an NFS-mounted file system.
-if df --local . >/dev/null 2>&1; then
-  (cd no-acce3s/d && chmod a-r . && chmod a-rx .. &&
-      mkdir -p a/b "$p/b/c" d/e && test -d a/b && test -d d/e) || fail=1
-  test -d "$p/b/c" || fail=1
-fi
+# XXX: This test fails with sydbox as well, see my comment in bug 204. -alip
+#if df --local . >/dev/null 2>&1; then
+#  (cd no-acce3s/d && chmod a-r . && chmod a-rx .. &&
+#      mkdir -p a/b "$p/b/c" d/e && test -d a/b && test -d d/e) || fail=1
+#  test -d "$p/b/c" || fail=1
+#fi
 
 b=`ls "$p/a" | tr -d '\n'`
 # With coreutils-5.3.0, this would fail with $b=bu.
